src/core/tracking/google/GoogleAnalytics.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * @flow
+ */
+
+const GOOGLE_TRACKING_ID :string = 'UA-118446829-3';
```

Comment:
Is this safe to check-in? This isn't like a secret or anything right?

Reply:
Correct. It's not a secret.

https://developers.google.com/analytics/devguides/collection/gtagjs/